tsconfig.json

```
@@ -0,0 +1,67 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "es2017",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
+    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    "lib": ["es2015", "dom"],                 /* Specify library files to be included in the compilation. */
```

Since this won't run in the browser:

```suggestion
    "lib": ["es2015"],                 /* Specify library files to be included in the compilation. */
```

(And I think even Node 10 supports more than ES2015, but I'm not sure.)